src/callbacks_step/analysis_surface_2d.jl

```diff
@@ -0,0 +1,144 @@
+# This file contains callbacks that are performed on the surface like computation of
+# surface forces
+
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+"""
+    AnalysisSurfaceIntegral{Semidiscretization, Variable}(semi, 
+                                                          boundary_symbol_or_boundary_symbols, 
+                                                          variable)
+
+    This struct is used to compute the surface integral of a quantity of interest `variable` alongside 
+    the boundary/boundaries associated with `boundary_symbol` or `boundary_symbols`.
+    For instance, this can be used to compute the lift or drag coefficient of e.g. an airfoil in 2D.
```

Comment:
```suggestion
    For instance, this can be used to compute the lift or drag coefficient of e.g. an airfoil in 2D.
    In that case `variable` is [`LiftCoefficient`](@ref) or [`DragCoefficient`](@ref), respectively.
```

We should provide the links once the coefficients have appropriate docstrings.